drivers/hmis/app/graphql/types/base_enum.rb

```diff
@@ -37,7 +37,7 @@ def self.with_enum_map(enum_map, prefix: '', prefix_description_with_key: true)
         member_values = yield member if block_given?
 
         # Ensure we are using DATA_NOT_COLLECTED key for 99s
-        member_values[:key] = 'DATA_NOT_COLLECTED' if member_values[:value] == 99
+        member_values[:key] = 'DATA_NOT_COLLECTED' if [99, '99'].include?(member_values[:value])
```

Comment:
Is there some way to avoid dealing with both ints and strings? Maybe we can transform everything to a string? Looks like it's only in two places here so not a big deal but I'd try to minimize this practice if possible

If that's not possible than alternatively we can could it inside the comparison:
```suggestion
        member_values[:key] = 'DATA_NOT_COLLECTED' if member_values[:value].to_s == '99'
```

Reply:
I'm not sure what the implication is going to be of changing all the Enum values into strings, I worry it might require a bunch of changes in the form processors. That does seem ideal, but just want to minimize impact given the time frame. So, just doing to do it for this specific case where the map contains a mix of strings and ints.